### DISCUSSION TITLE
New Admin

### DISCUSSION THREAD

**Discussion by s34m** [topgrade-rs/topgrade#498] [Announcements]
Herby @SteveLauC will be granted the Role Admin for this repository. Thanks for your work for this project <3

**Comment by SteveLauC** [comment]
Thanks:)

**Comment by s34m** [comment]
@SteveLauC you should now also have access to the crate.

**Reply by SteveLauC** [reply]
> @SteveLauC you should now also have access to the crate.

Yep, thank you!
